bigchaindb/__init__.py

```diff
@@ -59,6 +59,10 @@
         'workers': None,  # if none, the value will be cpu_count * 2 + 1
         'threads': None,  # if none, the value will be cpu_count * 2 + 1
     },
+    'wsserver': {
+        'host': os.environ.get('BIGCHAINDB_WSSERVER_HOST') or 'localhost',
```

Comment:
My *personal* preference would be to `s/wsserver/websocket/g`, little bit more readable, but that's personal.